indico/modules/events/abstracts/forms.py

```diff
@@ -212,7 +212,11 @@ class AbstractJudgmentFormBase(IndicoForm):
     accepted_track = QuerySelectField(_("Track"), [HiddenUnless('judgment', AbstractAction.accept)],
                                       get_label='title', allow_blank=True, blank_text=_("Choose a track..."),
                                       description=_("The abstract will be accepted in this track"))
```

Comment:
The PR is named "Abstracts bulk acceptance", what do you mean it's not related? Yes, it's functional without it, but the description texts are misleading.